src/model/api.ts

```diff
@@ -1,3 +1,8 @@
 import { FirebaseError } from 'firebase-admin';
 
 export type ApiError = Partial<Error & FirebaseError> & { msg: string };
+
+export interface ApiCallResult {
```

Comment:
Shouldn't this just be an intersection type?
```typescript
export type ApiCallResult = ApiError & { code?: number };
```

Reply:
Actually, you shouldn't even have to define this new type at all (see my other comments).